src/reconcile.py

```diff
@@ -207,13 +228,18 @@ def _server_only(table_row: types_.TableRow, discourse: Discourse) -> types_.Del
         raise exceptions.ReconcilliationError(
             f"internal error, expecting link on table row, {table_row=!r}"
         )
-    return types_.DeleteAction(
-        type_=types_.ActionType.DELETE,
-        level=table_row.level,
-        path=table_row.path,
-        navlink=table_row.navlink,
-        content=discourse.retrieve_topic(url=table_row.navlink.link),
-    )
+    try:
+        return types_.DeleteAction(
+            type_=types_.ActionType.DELETE,
+            level=table_row.level,
+            path=table_row.path,
+            navlink=table_row.navlink,
+            content=discourse.retrieve_topic(url=table_row.navlink.link),
+        )
+    except exceptions.DiscourseError as exc:
+        raise exceptions.ServerError(
+            f"failed to retrieve contents of page, url={table_row.navlink.link}"
+        ) from exc
```

Comment:
I'd rather enclose only the `retrieve_topic` call in the try/catch block
So something like
```
try:
   content = discourse.retrieve_topic(url=table_row.navlink.link)
except exceptions.DiscourseError as exc:
   ...
return  types_.DeleteAction(...)
```

Reply:
Yes that is better, should be fixed shortly

Reply:
Should be fixed now